### PR TITLE
Misc Tutorial Improvements, some for noetic

### PR DESCRIPTION
**Pull request by DLu**
### Description

As I've been going through the tutorials, I've been clarifying / fixing a few things as I go. This PR contains
 * Copyediting
 * Missing dependencies (at least for noetic)
 * ~Convert a webm to a gif to get it to embed properly.~

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Review by JafarAbdi — APPROVED** [review]

**Review by AndyZe — APPROVED** [review]

File: doc/trac_ik/trac_ik_tutorial.rst [hunk 1]
```diff
@@ -10,10 +10,12 @@ The package `trac_ik_kinematics_plugin <https://bitbucket.org/traclabs/trac_ik/s
 Install
 -------
 
-As of v1.4.5, **trac_ik** is part of the ROS Indigo/Jade binaries: ::
+As of v1.5.1, **trac_ik** is part of the ROS Kinetic/Melodic binaries: ::
 
   sudo apt-get install ros-melodic-trac-ik-kinematics-plugin
 
+Noetic Tracked (tracced?) here: https://bitbucket.org/traclabs/trac_ik/issues/67/ros-noetic-support
```
Comment (felixvd) [inline doc/trac_ik/trac_ik_tutorial.rst:17]: I'd either make this a full sentence or remove it.

File: README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ This repository is currently built automatically by two systems. Travis builds t
 
 ## Build Locally
 
-If you want to test the tutorials by generating the html pages locally on your machine, use the ``build_locally`` script.
+If you want to test the tutorials by generating the html pages locally on your machine, you will first need to install the `rosdoc_lite` package, and then you can use the ``build_locally`` script.
```
Comment (felixvd) [inline README.md:25]: Would be good to include the install commands.

File: doc/planning_scene_monitor/planning_scene_monitor_tutorial.rst [hunk 3]
```diff
@@ -5,11 +5,11 @@ The :planning_scene_monitor:`PlanningSceneMonitor` is the recommended interface
 
 RobotState
 ----------
-The :moveit_core:`RobotState` is a snapshot of a robot. It contains information about the geometry of the robot as well as a set of joint values.
+The :moveit_core:`RobotState` is a snapshot of a robot. It uses information from the :moveit_core:`RobotModel` and a set of joint values to examine the geometry of the robot.
```
[the remark this answers is not in the document]
Comment (DLu) [reply doc/planning_scene_monitor/planning_scene_monitor_tutorial.rst:8]: This tutorial comes two after [RobotModel and RobotState](https://ros-planning.github.io/moveit_tutorials/doc/robot_model_and_robot_state/robot_model_and_robot_state_tutorial.html) which makes a distinction between those two pieces, so I updated this bit to match the information from the other tutorial. 

Can you be more specific about what you dislike?

[the remark this answers is not in the document]
Comment (felixvd) [reply doc/planning_scene_monitor/planning_scene_monitor_tutorial.rst:8]: Personally I'm not a fan of writing that the `RobotState` examines the geometry of the robot. Saying that it contains a `RobotModel` and joint values would be enough.

[the remark this answers is not in the document]
Comment (DLu) [reply doc/planning_scene_monitor/planning_scene_monitor_tutorial.rst:8]:
```suggestion
The :moveit_core:`RobotState` is a snapshot of a robot. It contains the :moveit_core:`RobotModel` and a set of joint values.
```

[the remark this answers is not in the document]
Comment (felixvd) [reply doc/planning_scene_monitor/planning_scene_monitor_tutorial.rst:8]: Good with me at least 👍

[the remark this answers is not in the document]
Comment (DLu) [reply doc/planning_scene_monitor/planning_scene_monitor_tutorial.rst:8]: @AndyZe ?

[the remark this answers is not in the document]
Comment (AndyZe) [reply doc/planning_scene_monitor/planning_scene_monitor_tutorial.rst:8]: Sounds good!

**Review by felixvd — APPROVED** [review]

**Pull request merged**